tests/transports-governance/conftest.py

```
@@ -0,0 +1,668 @@
+    token_last_reset: Optional[str] = None
+    request_last_reset: Optional[str] = None
+
+
+@dataclass
+class TestCustomer(TestEntity):
+    """Test customer entity"""
+
+    name: str = ""
+    budget_id: Optional[str] = None
+    budget: Optional[TestBudget] = None
+    teams: Optional[List["TestTeam"]] = None
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)